config.yml

```
@@ -9,8 +9,8 @@ description: # a longer description for your Learning Lab course, displayed on t
 # This will be used when the repository is crated for each individual learner
 
 template:
-  name: # the name of the repository when it is created for the learner
-  repo: # the name of the template repository from which to clone for the learner
+working-with-github-learning-lab
```

Syntax mistake. It should really be:

```suggestion
  name: working-with-github-learning-lab
```